### DISCUSSION TITLE
airflow job stuck at queue state and get running after 2 mins

### DISCUSSION THREAD

**Discussion by haoplus** [apache/airflow#20674] [Q&A]
Hi 

I meet a very strange problem that if we trigger the airflow dag from external, the one will stay in queue for 2 minutes and then the first task go into "running" state. ("none" state before)  -- no any other jobs in the meantime,  the slot is 128. 

if the dag started by scheduled time, then it will immediate executed.  Or if we manually trigger the dag from UI, with "ignore xxx",  it will start immediate. 

![image](https://user-images.githubusercontent.com/9104393/148206254-3a4f9b68-81f6-4fb0-9a7b-a7bffa45c6fd.png)

I'm using airflow in a kubernetes environment with CeleryExecutor and redis as message queue.  We don't find limitation in CPU or memory usage when monitor it. 

Configuration: 

`
    [core]
    dags_folder = /opt/airflow/dags
    executor = CeleryExecutor
    sql_alchemy_conn = postgresql+psycopg2://...
    sql_alchemy_schema = public
    load_examples = False
    plugins_folder = /opt/airflow/plugins
    parallelism = 32
    max_active_tasks_per_dag = 20
    dags_are_paused_at_creation = True
    max_active_runs_per_dag = 20
    default_task_retries = 1
    sql_alchemy_pool_size = 16
    
    [scheduler]
    run_duration = 7200
    min_file_process_interval = 600
    catchup_by_default = False
    

    [logging]
    base_log_folder = /opt/airflow/logs
    remote_logging = False

    [api]
    auth_backend = airflow.api.auth.backend.basic_auth

    [webserver]
    base_url = https://0.0.0.0:8080
    default_ui_timezone = UTC
    web_server_host = 0.0.0.0
    web_server_port = 8080
    workers = 1
    enable_proxy_fix = True

    [celery_kubernetes_executor]
    kubernetes_queue = kubernetes

    [celery]
    celery_app_name = airflow.executors.celery_executor
    worker_concurrency = 100
    worker_log_server_port = 8793
    broker_url = redis://redis:6379/0
    result_backend = db+postgres://....

    flower_host = 0.0.0.0
    flower_url_prefix = /log-insight/flower
    flower_port = 5555

    [kubernetes]
    worker_container_repository =apache/airflow
    worker_container_tag = 2.2.1
    namespace = default
    delete_worker_pods = True
    delete_worker_pods_on_failure = False
    worker_pods_creation_batch_size = 20
    multi_namespace_mode = False
    in_cluster = True
    verify_ssl = False

The first task timestamp: 
![image](https://user-images.githubusercontent.com/9104393/148205924-20d4386e-97de-4d9f-ae15-6ef2b842441a.png)

the dags has these parameters may or may not relavant : 

![image](https://user-images.githubusercontent.com/9104393/148207220-3aa8afa5-3adf-41f2-83ff-488b00669828.png)

Would someone could help me to diagnosis this problem?  
`

**Comment by AAASIF** [comment]
I'm facing same issue , can anyone resolve it ?
Setting airflow.cfg
# provided explicitly or passed via ``default_args``
default_owner = airflow
default_cpus = 1
default_ram = 512
default_disk = 512
default_gpus = 0

# Default queue that tasks get assigned to and that worker listen on.
default_queue = default
